### ISSUE TITLE
View knitted R Markdown PDF documents in the Viewer Pane similar to Quarto rendering

### ISSUE THREAD

**Issue by chuxinyuan** [rstudio/rstudio#7129]
Every time I use R Markdown to generate a PDF document, I wonder why not display it in the Viewer? Every time I type code for a PDF e-book, I think how great it would be to display a PDF document in the Viewer while I type the code.

Maybe I need  a function view_pdf(),It can read a PDF file and browse in the Viewer.

**Comment by ronblum** [comment]
@chuxinyuan Thank you for filing the issue. We'll review it as we continue development of RStudio.

**Comment by ForceBru** [comment]
The issue is still present in RStudio 1.4.1103: the "Preview in Viewer Pane" setting does absolutely nothing, and the PDF opens in RStudio's own PDF viewer anyway.

**Comment by cdriveraus** [comment]
this would be particularly convenient for when working on rstudio server...

**Comment by chuxinyuan** [comment]
@ronblum The problem has been neglected for a long time.

**Comment by ronblum** [comment]
This feature was introduced for documents rendered by Quarto (qmd). This is then an enhancement request to also have it available for R Markdown documents.

**Comment by cderv** [comment]
@ronblum maybe this part of the dropdown menu should be removed when Rmd file when pdf output ? 
I don't know if that is part of the confusion, but just seen it while answering to a question related to this: 
https://community.rstudio.com/t/knitting-without-preview-word-and-pdf/146852/13

**Comment by SVNKoch** [comment]
yep it is very confusing, the UI suggests that this should be possible!

**Comment by chuxinyuan** [comment]
This problem has not been paid attention to for a long, long time, forget it, programmers develop her very hard.

**Comment by salim-b** [comment]
> This feature was introduced for documents rendered by Quarto (qmd). This is then an enhancement request to also have it available for R Markdown documents.

Note that this is set globally in `rstudio-prefs.json` via [`"rmd_viewer_type": "pane"`](https://docs.posit.co/ide/server-pro/reference/session_user_settings.html), not per document. Which is fine, but the UI is confusing.

**Comment by chuxinyuan** [comment]
It's been a long, long time since anyone has followed up on this issue.
